dashboard/test/controllers/home_controller_test.rb

```diff
@@ -362,7 +362,7 @@ class HomeControllerTest < ActionController::TestCase
   # TODO: remove this test when workshop_organizer is deprecated
   test 'workshop organizers see dashboard links' do
     sign_in create(:workshop_organizer, :with_terms_of_service)
-    query_count = 15
+    query_count = 14
```

Comment:
out of curiosity: did this change because of the removal of the nps survey in haml?

Reply:
Yes. Specifically the removal of `show_nps_survey?` in `dashboard/app/views/home/_homepage.html.haml`